app/views.py

```diff
@@ -1,110 +1,277 @@
-            abort(400)
-
-    user = Users(username, password)
-    if username in Users.user_db.keys():
-        if Users.user_db[username] == password:
-            g.user = user
-            session['user'] = g.user.username
-            return {'logged in': g.user.username}, status.HTTP_200_OK
-        else:
-            return "username/password incorrect", status.HTTP_401_UNAUTHORIZED
+        message = {"message": "You need to register first before you login"}
+        return message, status.HTTP_400_BAD_REQUEST
 
-    return "username/password incorrect", status.HTTP_401_UNAUTHORIZED
+    # user = Users(email, password)
+    if email in Users.user_db.keys():
+        print(Users.user_db[email])
+        if Users.user_db[email] == password:
+            session['user'] = email
+            return {
+                'message': 'you have succesfully been logged in'}, status.HTTP_200_OK
+        else:
+            raise exceptions.AuthenticationFailed()
+    raise exceptions.AuthenticationFailed()
 
 
-@app.route('/api/auth/logout', methods=['POST'])
+@app.route('/api/auth/logout/', methods=['POST'])
 def logout():
+    """User Logout endpoints
+        logs out a user
+    ---
+        tags:
+            - Bright Events API
+        responses:
+            200:
+                description: you have been logged out
+    """
     if 'user' not in session:
-        return "you have to login first"
-    else:
-        session.pop('user')
-        return "you have been logged out"
+        message = {"message": "you have to login first"}
+        return message, status.HTTP_401_UNAUTHORIZED
+    session.pop('user')
+    message = {"message": "you have been logged out"}
+    return message, status.HTTP_200_OK
 
-# app.route('/api/auth/reset-password', methods=['POST'])
-# def reset_password():
 
-@app.route('/api/token')
-def get_auth_token():
-    token = g.user.generate_auth_token()
-    return jsonify({'token': token.decode('ascii')})
+@app.route('/api/auth/reset-password/', methods=['POST'])
+def reset_password():
+    """Reset user Password endpoint
+        takes in a password and resets the password
+    ---
+        tags:
+          - Bright Events API
+        parameters:
+          - in: formData
+            name: password
+            required: true
+            description: The password you want to reset!
+            type: string
+        responses:
+          201:
+            description: Password updated successfully
+            schema:
+              $ref: '#/definitions/Task'
+            examples:
+                password: password
 
-def api_view(key):
     """
-    Handles how the data will be 
-    in the browsable api
-    """
-    return {
-        'name': Events.events_db[key],
-        'url': request.host_url.rstrip('/') + url_for('events_details', key=key)
-    }
+    if 'user' in session:
+        password = request.data.get('password')
+        Users.user_db['user'] = password
+        print(Users.user_db['user'])
```

Comment:
remove ```print``` statements that are meant to test functionality

Reply:
noted and removed